File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.cpp
class Solution {
public:
    vector<string> removeAnagrams(vector<string>& words) {
       vector<string>ans;
       int n=words.size();
       for(int i=0;i<n;i++){
        if(i==0){
            ans.push_back(words[i]);
        }
        else{
            string pre=words[i-1];
            string cur=words[i];
            sort(pre.begin(),pre.end());
            sort(cur.begin(),cur.end());
            if(cur!=pre){
                 ans.push_back(words[i]);

            }
        }

       }

       return ans;
    }
};

File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.java
import java.util.*;
class Solution {
    public List<String> removeAnagrams(String[] words) {
        ArrayList<String> ans=new ArrayList<>();
        for(int i=0;i<words.length;i++){
            if(i==0){
                ans.add(words[i]);
            }
            else{
                String pre=words[i-1];
                String cur=words[i];
                char[] ch1=pre.toCharArray();
                char[] ch2=cur.toCharArray();
                Arrays.sort(ch1);
                Arrays.sort(ch2);
                String a=new String(ch1);
                String b=new String(ch2);
                System.out.println(a);
                System.out.println(b);
                if (!a.equals(b)) {
                 ans.add(cur);
                 }
            }

        }
        return ans;
    }
}